HostAbroadTests/src/test/integrationTests/ASSearchImpTest.java

```diff
@@ -0,0 +1,90 @@
+package integrationTests;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+import javax.persistence.Persistence;
+import javax.persistence.Query;
+
+import org.junit.Test;
+
+import com.business.TUser;
+import com.business.User;
+import com.business.ASFactory.ASFactory;
+import com.business.ASSearch.*;
+
+public class ASSearchImpTest {
+	private ASSearchImp assearch;
+	
+	@Test
+	public void searchHostTest() { 
+		ArrayList<TUser> aux = new ArrayList<TUser>();
```

Comment:
I do not know how this is working but the idea of JPA is to use Entity pattern- in your case User. Please keep in mind that  that and change your aux array. Also I suggest you to name variables better cuz it is hard to follow your code this way.